ads/google/a4a/traffic-experiments.js

```diff
@@ -75,19 +80,24 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
   const isSetFromUrl = maybeSetExperimentFromUrl(win, element,
       experimentName, externalBranches.control,
       externalBranches.experiment, MANUAL_EXPERIMENT_ID);
-  const experimentInfo = {};
-  experimentInfo[experimentName] = internalBranches;
+  const experimentInfoMap = {};
+  experimentInfoMap[experimentName] = {
+    isTrafficEligible: () => { return true; },
```

Comment:
I think, like @keithwrightbos mentioned elsewhere, that this can just be `isTrafficEligible: () => true`